modules/smaatoBidAdapter.js

```diff
@@ -242,19 +246,26 @@ const converter = ortbConverter({
 
     if (request.site) {
       request.site.id = window.location.hostname
-      if (siteContent) {
-        request.site.content = siteContent;
+      if (content) {
+        request.site.content = content;
+      }
+      setPublisherId(request.site);
+    } else if (request.dooh) {
```

Comment:
This feels very fragile; I would reverse the conditional and always send a dooh request if a dooh object exists, instead of putting it in the else.

Reply:
But there should be only one of `app`, `site` or `dooh` present in a request as per ortb specification

Reply:
Right, I am saying, right now you will always choose site if both site and dooh exist. Do you prefer to always choose DOOH in that scenario and jettison the site object?

Reply:
We generally prefer `site` over `dooh`
I thought (might be confusing with prebid-server tho) that there's an internal check that prevents this kind of misconfiguration before it reaches the adapter code?